collections/_release_4_4/entry-general-editor-persistent-window-state.md

```diff
@@ -0,0 +1,20 @@
+---
+type: entry
+section: general
+subsection: editor
+rank: 7
+importance: 3
+anchor: persistent-window-state
+title: Persistent window state
+blockquote: Continue where you left off
+text: |
+  This is a quality of life improvement for anyone using big monitors or multi-monitor setups.
+
+  The Godot editor now keeps track of its window state: whether it is in fullscreen or windowed mode, the screen it is on, as well as size and position.
```

Comment:
```suggestion
  The Godot editor now keeps track of its window state: whether it is in fullscreen or windowed mode, the screen it is on, as well as its size and position.
```

Reply:
In addition, I suggest "which screen it is on" rather than "the screen it is on".